boutiques_descriptors/explore-asl.json

```diff
@@ -29,155 +31,49 @@
-        },
-        {
-            "id": "source_structure",
-            "name": "Source Structure JSON",
-            "description": "Config file defining structure of DICOM directory. Only required for DICOM. See ExploreASL documentation for details.\nNOTE: This field is not required if sourceStructure.json is included in the input folder (at root level).",
-            "optional": true,
-            "type": "File",
-            "value-key": "[SOURCE_STRUCTURE]",
-            "command-line-flag": "--source_structure"
-        },
-        {
-            "id": "study_par",
-            "name": "Study Parameter JSON",
-            "description": "Config file containing study description fields. Recommended for DICOM input. See ExploreASL documentation for details.",
-            "optional": true,
-            "type": "File",
-            "value-key": "[STUDY_PAR]",
-            "command-line-flag": "--study_par"
-        },
-        {
-            "id": "parameter_behaviour",
-            "name": "Parameter Behaviour",
-            "description": "If there is already a dataPar.json present in the input directory, do you wish to keep the original, overwrite, merge with priority to the original, or merge with priority to the new parameters?",
-            "optional": true,
-            "type": "String",
-            "value-choices": [
-                "Defer",
-                "Overwrite",
-                "Merge_to_original",
-                "Merge_to_new"
-            ],
-            "default-value": "Merge_to_new",
-            "value-key": "[PARAMETER_BEHAVIOUR]",
-            "command-line-flag": "--parameter_behaviour"
-        },
-        {
-            "id": "import_modules_dcm2nii",
-            "name": "DCM2NII Conversion",
-            "description": "Use the DICOM to NIfTI import module",
-            "optional": true,
-            "default-value": 0,
-            "value-choices": [
-                0,
-                1
-            ],
-            "type": "Number",
-            "value-key": "[ImportModules_DCM2NII]",
-            "command-line-flag": "--im_dcm2nii"
-        },
-        {
-            "id": "import_modules_nii2bids",
-            "name": "NII2BIDS Conversion",
-            "description": "Use the NIfTI to BIDS import module",
-            "optional": true,
-            "default-value": 0,
-            "value-choices": [
-                0,
-                1
-            ],
-            "type": "Number",
-            "value-key": "[ImportModules_NII2BIDS]",
-            "command-line-flag": "--im_nii2bids"
+            "command-line-flag": "--dataset_description_json"
         },
         {
             "id": "import_modules_deface",
             "name": "Defacing",
             "description": "Use the defacing import module",
-            "optional": true,
-            "default-value": 0,
             "value-choices": [
-                0,
-                1
+                "0",
```

Comment:
Why it became a string ? 

According to what we can found in [Boutiques](https://github.com/boutiques/boutiques/blob/5a2374e87dd837cb83fec9603f32e3c9085613ba/tools/python/boutiques/schema/examples/example1/example1_docker.json#L78).

Reply:
Yes, this will fail to validate, given the input type is set to 'Number' (line 51)